src/Components/GeneralComponents/GeneralWindow.css

```diff
@@ -1,3 +1,43 @@
-.generalWindowContentContainer {
+.generalWindow {
+  background-color: #ddcc66;
+  text-align: right;
+  padding: 7px;
+  border-radius: 5px;
+  z-index: 1;
+}
+
+.centered {
+  width: fit-content;
+  position: fixed;
+  top: 52%; /* simply wanted it down a bit for astetics, thats whats behind the 2%*/
+  left: 50%;
+  transform: translate(-50%, -50%);
+  /*  the transform makes the window's anchor point to be in it's center instead of top left,
+        that way it is possible to center it easily instead of calculating its position according
+        to it's dimensions   */
+}
+
+.rightSided {
+  width: fit-content;
+  position: fixed;
+  right: 100px;
+  top: 52%;
+  transform: translate(0, -50%);
+  /*  same case as the comment above, instead this time I wanted it's anchor point to be on
+        the center-right edge, so I can space from the right according to the existing buttons size in px  */
+}
+
+.generalWindowCloseButton {
+  margin-left: auto;
+}
+
+.contentContainer {
   margin-top: 7px;
+  background: #ffd095;
+  border-radius: 7px;
+  border: 1px #ddb075 solid;
+  text-align: center;
+  height: 80vh;
+  width: fit-content;
+  min-width: 20vw;
```

Comment:
please move width, min-width and height to the top, in this order, for consistency reasons.